server/src/main/java/org/elasticsearch/action/admin/cluster/stats/CCSTelemetrySnapshot.java

```diff
@@ -66,6 +65,7 @@ public final class CCSTelemetrySnapshot implements Writeable, ToXContentFragment
 
     private final Map<String, Long> clientCounts;
     private final Map<String, PerClusterCCSTelemetry> byRemoteCluster;
+    private boolean useMRT = true;
```

Comment:
Either here or on the setUseMRT can you put a comment what MRT means. Non-ccs developers probably don't know what that abbreviation means on first glance.